src/main/java/com/nihongo/NihonGoApi.java

```diff
@@ -1,14 +1,14 @@
-package com.jpn.lang;
+package com.nihongo;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-public class JpnLangApiDemo {
+public class NihonGoApi {
 
```

Comment:
Can you rename this class like this NihongoAPI? this looks better

Reply:
i have changed